Makefile

```
@@ -6,14 +6,17 @@ ifneq (,$(DAPPER_HOST_ARCH))
 
 include $(SHIPYARD_DIR)/Makefile.inc
 
-TARGETS := $(shell ls -p scripts | grep -v -e /)
+TARGETS := $(shell ls -p scripts | grep -v -e / | grep -v reload-images)
 CLUSTERS_ARGS = --cluster_settings $(DAPPER_SOURCE)/scripts/kind-e2e/cluster_settings
 
 clusters: build images
 
 e2e: deploy
 	./scripts/kind-e2e/e2e.sh --deploytool $(deploytool)
 
+reload-images: build images
+	./scripts/$@ --globalnet true
```

I think "--globalnet true" is not required for this target. Can you please check.

---

Oh true let me fix that 
```suggestion
	./scripts/$@
```